src/Microphysics/saturation_adjustment.jl

```diff
@@ -68,6 +73,7 @@ Return `WarmPhaseEquilibrium` representing an equilibrium between water vapor an
 """
 struct WarmPhaseEquilibrium <: AbstractEquilibrium end
 @inline equilibrated_surface(::WarmPhaseEquilibrium, T) = PlanarLiquidSurface()
+convert_equilibrium(FT, ::WarmPhaseEquilibrium) = WarmPhaseEquilibrium()
```

Comment:
Is this method necessary?  I presume this is already covered by https://github.com/NumericalEarth/Breeze.jl/blob/aa22c7b13cbe1b6e0a58f88a02934dc99fa1ddce/src/Microphysics/saturation_adjustment.jl#L63?